roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -37,3 +37,52 @@
 - set_fact:
     docker_options: "{{ openshift.docker.options | default(omit) }}"
   when: not openshift.docker.hosted_registry_insecure | default(False) | bool
+
+# Avoid docker 1.9 when installing origin < 1.2 or OSE < 3.2 on RHEL/Centos and
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=1304038
+- name: Default to latest docker
+  set_fact:
+    docker_version: ''
+    docker_downgrade: False
+  
+- name: Gather installed version of common package
+  shell: >
+    rpm -q {{ openshift.common.service_type }} --queryformat '%{version}' | cut -d '.' -f1,2 | grep -v 'not installed'
```

Comment:
this can be `rpm --quiet -q {{ openshift.common.service_type }} --queryformat '%{version}'` if using the version_compare filter